src/earth/earth.c

```diff
@@ -545,8 +548,14 @@ earthPreparePaintScreen (CompScreen *s,
 				((float)currenttime->tm_yday + 10.0f));
 	es->gha = (float)currenttime->tm_hour-(es->tz +
 				(float)currenttime->tm_isdst) +
-				(float)currenttime->tm_min / 60.0f;
-
+				(float)currenttime->tm_sec/3600.0000f;
+	
+	/* Animation */
+            es->rotation += es->rotspeed;
```

Comment:
Everything looks good except this should *not* be indented:
`es->rotation += es->rotspeed;`

Reply:
ok

Reply:
Looks good to me, thanks.